components/ErrorEventsDashboard.js

```diff
@@ -0,0 +1,142 @@
+import moment from 'moment'
+import { Component } from 'react'
+import { withAuth } from 'use-auth0-hooks'
+
+import { secureFetch } from '../util'
+
+class ErrorEventsDashboard extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      events: null,
+      eventsError: null
+    }
+    // TODO fix babel plugin so we can use class properties
+    // https://babeljs.io/docs/en/babel-plugin-proposal-class-properties
+    // https://nextjs.org/docs/advanced-features/customizing-babel-config
+    this.handleFetchErrors = this.handleFetchErrors.bind(this)
+  }
+
+  async handleFetchErrors (force = false) {
+    const { events, eventsError } = this.state
+    if (!force && (events || eventsError)) {
+      return
+    }
+
+    const { accessToken } = this.props.auth
+    if (!accessToken) {
+      return
+    }
+    const url = `${process.env.API_BASE_URL}/api/admin/bugsnag/eventsummary`
+    const fetchedErrorEvents = await secureFetch(url, accessToken)
+    if (fetchedErrorEvents) {
+      this.setState({
+        events: fetchedErrorEvents,
+        fetchMessage: `Updated at ${moment().format('h:mm:ss a')}`
+      })
+    } else {
+      this.setState({
+        fetchMessage: `Failed to update!`
+      })
+    }
+  }
+
+  async componentDidMount () {
+    await this.handleFetchErrors()
+  }
+
+  async componentDidUpdate () {
+    await this.handleFetchErrors()
+  }
+
+  render () {
+    const { auth } = this.props
+    const { events, eventsError, fetchMessage } = this.state
+    if (!auth.isAuthenticated) return null
+    const hasEvents = events && events.length > 0
+    return (
+      <div>
+        <h2>Error Events Summary</h2>
+        <div className='controls'>
+          <button onClick={this.handleFetchErrors}>
+            Fetch errors
+          </button>
+          {fetchMessage && <span className='fetchMessage'>{fetchMessage}</span>}
+          <a
+            className='push'
+            target='_blank'
+            rel='noopener noreferrer'
+            href='https://app.bugsnag.com/'
+          >
+            Open Bugsnag console
+          </a>
+        </div>
+        {hasEvents
+          ? (
+            <div>
+              {eventsError && <pre>Error loading events: {eventsError}</pre>}
+              <p>
+                All error events recorded over the last two weeks:
+              </p>
+              <table>
```

Comment:
I think it'd be better to figure out a standard paginating table component to use for this and other tabular data items throughout the app.

Reply:
I tend to prefer tabs also... I defer all layout resolutions to a review by a UX person.

Reply:
I think we should handle using a standard table component in a separate refactor/refinement PR once we have a better sense for all of the data elements we're interested in showing.